app/controllers/boards_controller.rb

```diff
@@ -13,22 +13,29 @@ def new
   end
 
   def create
```

Comment:
Could you please add specs

Reply:
You asked for specs, I gave you specs)))